### ISSUE TITLE
[bug #21432] Incorrect timer interrupt handler names for ATtiny45

### ISSUE THREAD

**Issue by avrs-admin** [avrdudes/avr-libc#261]
Thu 25 Oct 2007 08:47:19 PM CEST

According to the ATtiny45 datasheet and the avr libc documentation, timer interrupt vector names should be:
TIMER1_COMPA_vect
TIMER1_OVF1_vect
TIMER1_OVF0_vect
instead of TIM1_..._vect in include/avr/iotnx5.h

This issue was migrated from https://savannah.nongnu.org/bugs/?21432

**Comment by avrs-admin** [comment]
Joerg Wunsch <joerg_wunsch>
Thu 25 Oct 2007 09:20:51 PM CEST

Confirmed.  That appears to be a recent change.  My older
datasheet used the names we've got in our header file, and
even the XML files from AVR Studio 4.13 b528 have it that
way.  I hope they've fixed the XML meanwhile...

It appears that change was introduced in datasheet Rev.
2586D-02/06, at least that's the only entry that mentions
changes to table 10-1 (without telling **what** has been
changed, sigh).

For the sake of backwards compatibility, we should then have
both names.
